Bookish/Models/ListOfBooks.cs

```diff
@@ -2,5 +2,6 @@ namespace Bookish.Models;
 
 public class ListOfBooks
```

Comment:
If this model is to use in a view, I'd create a new folder called `ViewModels` and put this in there